### PR TITLE
Fix ASR Webhook Description

### PR DESCRIPTION
**Pull request by vshisterov**
## Description

Input event callback payload structure adjusted.

## Deploy Notes

Notes regarding deployment the contained body of work. These should note any db migrations, etc.

### CODE REVIEW

File: _documentation/en/voice/voice-api/webhook-reference.md [hunk 1]
```diff
@@ -360,6 +382,5 @@ The event endpoint will also receive webhooks in the event of an error. This can
 Field | Example | Description
  -- | -- | --
 `reason` | `Syntax error in NCCO. Invalid value type or action.` | Information about the nature of the error
-`error` | `ERR1: Failed to analyze audio` | This field present for automatic speech recognition
```
Comment (benhayehudi) [inline _documentation/en/voice/voice-api/webhook-reference.md:363]: Why are we removing this `error` field from the example? Does it no longer exist?

File: _documentation/en/voice/voice-api/webhook-reference.md [hunk 2]
```diff
@@ -307,13 +307,35 @@ Field | Example | Description
...
+Field | Example | Description
+ -- | -- | --
+`digits` | `42` | The buttons pressed by the user 
+`timed_out` | `true` | Whether the DTMF input timed out: `true` if it did, `false` if not
+
+#### Speech Recognition Results
+
+Field | Example | Description
+-- | -- | --
+`timeout_reason` | `end_on_silence_timeout` | Indicates if the input ended when user stopped speaking (end_on_silence_timeout), by max duration timeout (max_duration) or if the user didn't say anything (start_timeout)
```
Comment (benhayehudi) [inline _documentation/en/voice/voice-api/webhook-reference.md:327]:
```suggestion
`timeout_reason` | `end_on_silence_timeout` | Indicates if the input ended when user stopped speaking (`end_on_silence_timeout`), by max duration timeout (`max_duration`) or if the user didn't say anything (`start_timeout`)
```

File: _documentation/en/voice/voice-api/webhook-reference.md (hunk 1 above)
Comment (vshisterov) [reply _documentation/en/voice/voice-api/webhook-reference.md:363]: Anyway it's not in the root of the payload, it's nested to `speech` object:
```
{
    "speech": {
        "error": "Speech was not enabled"
    }, ...
```
so now it's confusing. Also, it's inconsistent with other "error callbacks" (with `reason` param) which should be someday also covered in more details.

Let's keep it in `input` callback description (which I adjusted in this PR).

Comment (benhayehudi) [reply _documentation/en/voice/voice-api/webhook-reference.md:363]: Okay, I'm fine with that, just as long as we document it somewhere. I believe our new orientation should be every response in a payload that's discovered should be documented in our platform. We need to account for developers working in strongly typed paradigms, where these things unaccounted for can throw them off.

**Pull request merged**